### DISCUSSION TITLE
Parent ID of Result Combination

### DISCUSSION THREAD

**Discussion by NathanMSY** [Dlubal-Software/Dlubal_CSharp_Client#60] [Q&A] [answered]
I want to calculate only the result combinations in my model with the "calculate_specific" method.
the method accepts an array of "calculate_specific_loading" objects.

a "calculate_specific_loading" object consists of the properties "type" and "no", with the type being the type (load case, load combination etc.).

When I use the load case or load combination type "E_OBJECT_TYPE_LOAD_CASE" or "E_OBJECT_TYPE_LOAD_COMBINATION" the request works and the corresponding cases/combinations are calculated.

When I use the type "E_OBJECT_TYPE_RESULT_COMBINATION" and the no 1 (with my model having a valid result combination with number 1), the service returns a calculation_fault with the error message: "There is no Result_Combination with ID 1 and parent ID %2."

REQUEST:
![image](https://github.com/Dlubal-Software/Dlubal_CSharp_Client/assets/75130759/547105d2-c42a-4dd1-8b1b-bdc5c239565f)

RESPONSE:
![image](https://github.com/Dlubal-Software/Dlubal_CSharp_Client/assets/75130759/4edd980d-e794-4b39-b599-0ebbcbf45738)

The ID in the error message corresponds with the "no"-property. What is the parent ID supposed to represent and how can I set the parent ID? the calculate_specific_loading object has only a "type" and a "no" property, no "parent_no" -property.

How can I calculate the result combination with number 1?

**Comment by Rebecca-Coding21** [comment, marked as answer]
Hi @NathanMSY,
I could reproduce the described problem. The parent id can't be defined for this function. We will fix this as soon as possible. The internal bug id is: 240430.
